sys/include/xtimer.h

```diff
@@ -427,17 +427,22 @@ static inline bool xtimer_less(xtimer_ticks32_t a, xtimer_ticks32_t b);
 static inline bool xtimer_less64(xtimer_ticks64_t a, xtimer_ticks64_t b);
 
 /**
- * @brief lock a mutex but with timeout
+ * @brief try to lock a mutex but with timeout
  *
- * @note this requires core_thread_flags to be enabled
+ * This will try to lock a mutex. If the mutex is not available immediately or
+ * until a certain amount of time (timeout) the method will return -1
```

Comment:
I think your sentence is in this case misleading, it's not clear if the mutex will be locked only during `timeout` or if it will try to lock it during the `timeout`. I will use the `@p` in any case :-)